### DISCUSSION TITLE
"is not annotated as @Input" simlink on macOS android

### DISCUSSION THREAD

**Discussion by Alexufo** [callstack/react-native-builder-bob#476] [Q&A] [answered]
### Description

I have an module with android activity.

All works fine when we working with example folder inside module. But on MacOS after npm install ../module we get 
![image](https://github.com/callstack/react-native-builder-bob/assets/1562329/f1f41f14-f09f-4817-a4d2-0f3fe6c6d674)

In Android studio on building process I get "is not annotated as @Input". But if we copy 'module'  directly to 'node_modules' - error will go away.

I found some detailes https://github.com/stripe/stripe-react-native/issues/1078#issuecomment-1372878878

It reproduced only on MacOS. The same module installed with 'npm install' compile on WindowsPC sucsessfully.

I suppose it is android plugin gradle plugin issue?

### Packages

- [X] create-react-native-library
- [X] react-native-builder-bob

### Selected options

-

### Environment

MacOs

**Comment by satya164** [comment, marked as answer]
We don't recommend doing `npm install ../module` to test the package:
https://callstack.github.io/react-native-builder-bob/faq#how-to-test-the-library-in-an-app-locally

**Reply by Alexufo** [reply]
But how do I integrate into the project and continue development?

**Reply by satya164** [reply]
You should publish to a private registry if you want to share your library with your other projects.

If you don't want to publish, you can use a monorepo or a local library within your repo: https://callstack.github.io/react-native-builder-bob/create#local-library
